jekyll/_cci2/caching.md

```diff
@@ -189,6 +189,22 @@ neither `save_cache` nor `restore_cache` support globbing for the `paths` key.
 
 ## Using Keys and Templates
 
+A cache-key is a _user-defined_ string that corresponds to a data cache. A cache-key can be created by interpolating **dynamic values** — these are called **templates**. Anything you see in a cache-key between curly braces is a template. Consider the following example:
+
+```sh
+{% raw %}myapp-{{ checksum "package-lock.json" }}{% endraw %}
+```
+
+The above example will output a unique string to represent this key. Here, the example is using a [checksum](https://en.wikipedia.org/wiki/Checksum) to create a unique string that represents the contents of a `package-lock.json` file.
+
+So, for the sake of demonstration, the above example might put out a string that looks like so:
```

Comment:
```suggestion
The example may output a string that looks like the following:
```

Reply:
just making this a bit simpler for translation ('for the sake of' is colloquial and harder to translate)